packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -4,7 +4,8 @@ import {Event as RawEvent} from './call-diagnostic/generated-types-temp/Event';
 export type SubmitClientEventOptions = {
   meetingId?: string;
   mediaConnections?: any[];
-  error?: any;
+  rawError?: any;
+  parsedError?: any;
```

Comment:
Can you use `ClientEventError` instead of `any`

Reply:
I can't because that's a type that we receive from the server. ClientEventError is the final type that we need to send.

Reply:
if `parsedError` is the error from the server then what is the `rawError`

Reply:
In that case can't we create a separate type for what we receive from the server?

Reply:
AAA, sorry, i thought you were referring to rawError, my bad

Reply:
fixed

Reply:
@mkesavan13, this could be either locus or meetinginfo, and potentially more in the future. I don't think it's worth it.

Reply:
well, i've removed it.. as it was useless